src/pages/register-a-beacon/land-communications.tsx

```diff
@@ -0,0 +1,247 @@
+            id="portableVhfRadioInput"
+            label="Portable MMSI number"
+            hintText="This is the unique MMSI number associated to the portable radio and is 9 numbers long. E.g. starts with 2359xxxxx"
+            defaultValue={form.fields.portableVhfRadioInput.value}
+          />
+        </FormGroup>
+      </CheckboxListItem>
+      <CheckboxListItem
+        id="satelliteTelephone"
+        value={Communication.SATELLITE_TELEPHONE}
+        defaultChecked={
+          form.fields.satelliteTelephone.value ===
+          Communication.SATELLITE_TELEPHONE
+        }
+        label="Satellite Telephone"
+        conditional={true}
+      >
+        <FormGroup
+          errorMessages={form.fields.satelliteTelephoneInput.errorMessages}
+        >
+          <Input
+            id="satelliteTelephoneInput"
+            label="Enter phone number"
+            hintText="Iridium usually start: +8707, Thuraya usually start: +8821, Globalstar usually start: +3364)"
+            defaultValue={form.fields.satelliteTelephoneInput.value}
+          />
+        </FormGroup>
+      </CheckboxListItem>
+      <CheckboxListItem
+        id="mobileTelephone"
+        value={Communication.MOBILE_TELEPHONE}
+        defaultChecked={
+          form.fields.mobileTelephone.value === Communication.MOBILE_TELEPHONE
+        }
+        label="Mobile Telephone(s)"
+        conditional={true}
+      >
+        <FormGroup
+          errorMessages={form.fields.mobileTelephoneInput1.errorMessages}
+        >
+          <Input
+            id="mobileTelephoneInput1"
+            label="Mobile number 1"
+            inputClassName="govuk-!-margin-bottom-4"
+            defaultValue={form.fields.mobileTelephoneInput1.value}
+            htmlAttributes={{ autoComplete: "tel" }}
+          />
+        </FormGroup>
+
+        <Input
+          id="mobileTelephoneInput2"
+          label="Mobile number 2 (optional)"
+          defaultValue={form.fields.mobileTelephoneInput2.value}
+          htmlAttributes={{ autoComplete: "tel" }}
+        />
+      </CheckboxListItem>
+      <CheckboxListItem
+        id="otherCommunication"
+        value={Communication.OTHER}
+        defaultChecked={
+          form.fields.otherCommunication.value === Communication.OTHER
+        }
+        label="Other"
+        conditional={true}
+      >
+        <FormGroup
+          errorMessages={form.fields.otherCommunicationInput.errorMessages}
+        >
+          <TextareaCharacterCount
+            id="otherCommunicationInput"
+            label="Please provide details of how we can contact you"
+            defaultValue={form.fields.otherCommunicationInput.value}
+            maxCharacters={250}
+          />
+        </FormGroup>
+      </CheckboxListItem>
+    </CheckboxList>
+  </FormGroup>
+);
+
+export const getServerSideProps: GetServerSideProps = handlePageRequest(
+  "/register-a-beacon/more-details",
+  definePageForm
+);
+
+export default LandOtherCommunications;
```

Comment:
```suggestion
export default LandCommunications;
```
Same update as near the top of the file!